src/components/Slider/Slider.js

```diff
@@ -1,8 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Slider.scss';
 
-const Slider = props => {
-  return <>Slider</>;
+const Slider = () => {
+  const [move, setMove] = useState(0);
+
+  const moveRight = () => {
+    if (move >= -800) {
+      setMove(move - 350);
+    }
+  };
+
+  const moveLeft = () => {
+    if (move <= -350) {
+      setMove(move + 350);
+    }
+  };
```

Comment:
```suggestion
  const moveRight = () => {
    if (move >= -800) {
      setMove(move => move - 350);
    }
  };

  const moveLeft = () => {
    if (move <= -350) {
      setMove(move => move + 350);
    }
  };
```
이렇게 함수형 업데이트를 사용해 주세요!
혹시 함수형 업데이트에 대해 궁금하시다면 저한테 말씀해주세요~

Reply:
함수형 업데이트로 수정 완료했습니다